### DISCUSSION TITLE
Request for RainbowKit integration: Best Wallet

### DISCUSSION THREAD

**Discussion by BestWeb3** [rainbow-me/rainbowkit#2105] [Q&A] [answered]
Hi, we're Best Wallet. 

We're interested in integrating with RainbowKit, ultimately so that Best Wallet could be enabled as one of the available choices when connecting a wallet on sites like [DEX Screener](https://github.com/user-attachments/assets/93a27f87-0219-4d7a-9cd6-b406b5aeebb8). Our understanding is that this would be different to simply being added as a Custom Wallet.

Is anyone able to assist us in this process?

Thanks in advance.

**Links**
[Bestwallet.com](https://bestwallet.com/en)
[App Store](https://apps.apple.com/gb/app/best-wallet-buy-sell-crypto/id6451312105)
[Google Play](https://play.google.com/store/apps/details?id=com.bestwallet.mobile&hl=en)

**Comment by DanielSinclair** [comment, marked as answer]
You can reference the Custom Wallet docs [here](https://www.rainbowkit.com/docs/custom-wallets)

**Reply by BestWeb3** [reply]
Hi @DanielSinclair - what we're looking for is more similar to [this](https://github.com/rainbow-me/rainbowkit/discussions/1988) request. Based on that thread, it looks like we'd write the code ourselves and raise a PR for it?

**Comment by BestWeb3** [comment]
Hi RainbowKit Team,

We have raised a PR for the integration of our BestWallet feature into RainbowWallet.

Could you kindly review and merge the PR at your earliest convenience? We are excited to contribute to this great project!
Thank you for your time and consideration.

[Pull request](https://github.com/rainbow-me/rainbowkit/pull/2193)
